src/utils/tokens/pair.js

```diff
@@ -50,19 +51,62 @@ const price = (tradeHistoryEntry, baseToken, quoteToken) => {
   return price;
 };
 
-const formatPrice =
-  (price) =>
-    price ? price.toFormat(PRICE_DECIMAL) : null;
+const getBaseAndQuotePrice = (tradeHistoryEntry, baseToken, quoteToken) => {
 
+  if (tradeHistoryEntry.buyWhichToken === quoteToken && tradeHistoryEntry.sellWhichToken === baseToken) {
+    return {
+      basePrice: new BigNumber(tradeHistoryEntry.sellHowMuch),
+      quotePrice: new BigNumber(tradeHistoryEntry.buyHowMuch),
+    };
+  } else if (tradeHistoryEntry.buyWhichToken === baseToken && tradeHistoryEntry.sellWhichToken === quoteToken) {
+    return {
+      basePrice: new BigNumber(tradeHistoryEntry.buyHowMuch),
+      quotePrice:new BigNumber(tradeHistoryEntry.sellHowMuch),
+    };
+  }
+
+};
+
+const formatPrice = (price, fromWei = false, unit = ETH_UNIT_ETHER) => {
```

Comment:
`ETH_UNIT_ETHER` what does that suppose to mean ?

Reply:
as the name suggests its the unit, one of:

export const ETH_UNIT_WEI = 'wei';               // wei 	1 wei 	1
export const ETH_UNIT_KWEI = 'kwei';             // Kwei (babbage) 	1e3 wei 	1,000
export const ETH_UNIT_MWEI = 'mwei';             // Mwei (lovelace) 	1e6 wei 	1,000,000
export const ETH_UNIT_GWEI = 'gwei';             // Gwei (shannon) 	1e9 wei 	1,000,000,000
export const ETH_UNIT_MICROETHER = 'microether'; // microether (szabo) 	1e12 wei 	1,000,000,000,000
export const ETH_UNIT_MILIETHER = 'milliether';  // milliether (finney) 	1e15 wei 	1,000,000,000,000,000
export const ETH_UNIT_ETHER = 'ether';  // ether 	1e18 wei 	1,000,000,000,000,000,000